central/views/imagecve/db_response.go

```diff
@@ -83,6 +83,10 @@ type imageResponse struct {
 	ImageID string `db:"image_sha"`
 }
 
+type imageV2Response struct {
+	ImageID string `db:"id"`
```

Comment:
I think it should probably be `image_id` . The way these tags are generated is you take the search field on the proto and convert it to snake_case. In this case the search field is `Image ID` in the proto.

https://github.com/stackrox/stackrox/blob/master/proto/storage/image_v2.proto#L13